cylc/flow/cfgspec/workflow.py

```diff
@@ -196,24 +200,25 @@ def get_script_common_text(this: str, example: Optional[str] = None):
             ``False`` after finishing the :cylc:conf:`flow.cylc[runtime]`
             section.
 
-            In :ref:`Cylc 7 backward compatibility mode <Cylc_7_compat_mode>`,
-            implicit tasks are still allowed unless you explicitly set
-            this to ``False``.
+            .. admonition:: Cylc 7 compatibility mode
+
+               In :ref:`Cylc_7_compat_mode`, implicit tasks are still
```

Comment:
Have I missed a discussion about this?
There's a slight inconsistency in that I thought that we were going to allow implicit tasks in compatibility mode, unless a rose-suite.conf is present (since Rose 2019 had behaviour equivalent to `implicit tasks = False`).

Reply:
Good point, that was never updated in the docs